### PR TITLE
Update backmatter intro for 1.6

### PR DESCRIPTION
**Pull request by jdinan**

### CODE REVIEW

**Review by lstewart — APPROVED** [review]
Looks fine to me

**Comment by jdinan** [comment]
Closes #446

**Review by lstewart — APPROVED** [review]
Great that resolves the #446 issue

File: content/backmatter.tex [hunk 1]
```diff
@@ -667,9 +667,11 @@ \chapter{Changes to this Document}\label{sec:changelog}
 
 \section{Version 1.6}
 \label{changelog:v1.6}
-Major changes in \openshmem[1.6] include the addition of the new
-\FUNC{shmem\_team\_ptr}, \FUNC{shmem\_ibget}, and \FUNC{shmem\_ibput}
-functions.
+Major changes in \openshmem[1.6] include the addition of inclusive and
+exclusive scan operations, session routines, support for multiple
```
Comment (davidozog) [inline content/backmatter.tex:671]: Perhaps?:
```suggestion
exclusive scan collective operations, session routines, support for multiple
```

File: content/backmatter.tex [hunk 2]
```diff
@@ -667,9 +667,11 @@ \chapter{Changes to this Document}\label{sec:changelog}
...
+initialization and finalization calls, interleaved block transfer APIs,
+signal add and set APIs, and the
+\FUNC{shmem\_team\_ptr} function.
```
Comment (davidozog) [inline content/backmatter.tex:674]: Perhaps `shmem_team_ptr` is not a "major change"?  It's at least included below...

**Review by davidozog — APPROVED** [review]
Looks great, nice catch @jdinan.

I included a few low-stake nitpicks for your consideration ;)  I'm good with it as is though.

**Comment by lstewart** [comment]
Dave's changes look OK, but we're in the weeds and should stop and ship it.

**Comment by davidozog** [comment]
> Dave's changes look OK, but we're in the weeds and should stop and ship it.

Funny, I was just thinking a similar thing; and I'm not completely confident with the nits anyway.  I've retracted the suggestions, @jdinan please feel free to ping if curious, but truly this is good to go.

File: content/backmatter.tex (hunk 2 above)
Comment (jdinan) [reply content/backmatter.tex:674]: It was already there, so I'd rather not remove it at this stage..

**Pull request merged**